backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1808,7 +1811,7 @@ class PreviewInfoSchema(marshmallow.Schema):
     )
 
 
-class FileContentSchema(ContentSchema):
+class MessageFileContentSchem(ContentSchema):
```

Comment:
Is it a typo/mistake that is `Schem` and not `Schema`?

Reply:
yes ...

Reply:
fixed